src/test/java/unit/clients/impl/GetCreateEntryResultSumary.java

```diff
@@ -9,7 +9,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-class ApiClientUtilsTest {
```

Comment:
Can you put all these new unit tests in a folder called `ApiClientUtils` so that it is easy to tell when using the file explorer that these tests are for this class

Reply:
ok done